pkg/operator/operator_test.go

```diff
@@ -0,0 +1,229 @@
+package operator
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	osconfigv1 "github.com/openshift/api/config/v1"
+	osclientfake "github.com/openshift/client-go/config/clientset/versioned/fake"
+
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes/fake"
+	core "k8s.io/client-go/testing"
+	"k8s.io/client-go/tools/record"
+)
+
+var (
+	knownDate   = metav1.Time{Time: time.Date(1985, 06, 03, 0, 0, 0, 0, time.Local)}
```

Comment:
where is `knownDate` used?

Reply:
Thanks, missed it, remaining from copy-paste